generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
Haha, nope I was actually saying we **had** to use long here. I tried replacing `long` with `int` on a local branch, and generated test was nothing but throwing exceptions for every case because the Expected property was never true. The JSON parser parses every number (when using `object`) to `long`

I mostly pointed it out since I wasn't aware, and maybe others weren't as well :)